packages/next-on-pages/README.md

```diff
@@ -0,0 +1,3 @@
+# `@cloudflare/next-on-pages`
```

Comment:
I think that npm will grab the README from here rather than the root of the monorepo.
I wonder if we should move the useful content about this package here?

Reply:
yeah, thanks a lot for the comment this is exactly what I was wondering about

Would you like us to have the next-on-pages details here? but then what do we have in the root readme? just basic info and links to the packages readmes?

I wasn't sure about that since the repo would be a monorepo but in practice its still very much just for next-on-pages, so ideally I think it would be nice to have info there as well...

Basically I am thinking about people visiting the repository, I think ideally they should see all the various next-on-pages and getting started steps without having to go to that specific package... (and we'd definitely not want to duplicate the info...)

I'm a bit conflicted on this one (since all solution have some drawbacks unfortunately), but I think I'll give the above (basic info at the root) version a shot 🤔

Reply:
Hmm...

So the Angular monorepo goes the other way and directs people back to the monorepo root for the README: https://www.npmjs.com/package/@angular/core

Remix points you at their docs website: https://www.npmjs.com/package/remix as does React.

So perhaps it is not so bad as it is - as long as there is a clear link from the package's README to the monorepo README.

Reply:
I've updated the two READMEs, @petebacondarwin and @GregBrimble please have a look and let me know what you think 🙂 🙏